### ISSUE TITLE
Pull Main out of section

### ISSUE THREAD

**Issue by lesliedoherty** [lesliedoherty/portfolio#14]
# Why
The main element must not appear as a descendant of the section element.

# Solution
<main role="main" aria-labelledby="Label" class="content"> should be above a section element
